frontend/packages/core/src/button.tsx

```diff
@@ -174,12 +175,11 @@ const StyledIconButton = styled(MuiIconButton)<{
   ...colorCss(props.$palette),
 }));
 
-// TODO: (jslaughter) Update when large sizing is available with material-ui@5
-export interface IconButtonProps extends Pick<MuiIconButtonProps, "disabled" | "type" | "onClick"> {
+export interface IconButtonProps
+  extends Pick<MuiIconButtonProps, "disabled" | "type" | "onClick" | "size"> {
   /** The button variantion. Defaults to primary. */
   variant?: ButtonVariant;
   children: React.ReactElement;
-  size?: IconButtonSize;
```

Comment:
should we delete the type reference as well?

Reply:
I left it for the moment since it's being used in other spots. Wanted to keep it simple initially.